MAINTAINERS.md

```diff
@@ -47,8 +45,8 @@ contributions, we have a strong foundation to build on.
 | Ruxandra Gabriela Rosu | ruxandra-gabriela.rosu@ing.com |
 | Cong Chen | cong.chen@ing.com |
 | Wu Yue Feng | wyf1116@gmail.com |
-
-
+| Dimitrios Maimaris | dimi.maim@gmail.com |
```

Comment:
We should also remove their references in CODEOWNERS.md which governs how issues/reviews get assigned.

Either delete the id if others are present, the whole line if they are the only one, or replace with new owner

@mandy-chessell Also need to consider if any change to the ODPi maintainer groups are required